index.html

```diff
@@ -64,8 +64,37 @@
     <h2>
       Introduction
     </h2>
+    <p>
+      This document defines an API enabling a website to request presentation of a [=digital credential=].
+    </p>
+    <p>
+      The API design is agnostic to both credential presentation exchange protocols and credential formats. However, to promote interoperability this document includes a [[[#protocol-registry]]].
+    </p>
+    <p>
+      The API is designed to support the following goals:
+      <ul>
+        <li>
+          Keep the act of requesting separate from the specific protocol, thereby enabling flexibility and adaptability in both the protocol and credential formats. This way, the pace of change in browsers won't hinder progress nor block new developments.
+        </li>
+        <li>
+          Require request transparency, enabling user-agent inspection for risk analysis.
+        </li>
+        <li>
+          Assume response opacity (encrypted responses), enabling verifiers and holders to control where potentially sensitive personally identifiable information is exposed.
+        </li>
+        <li>
+          Forbid websites from accessing any information without user interaction, ensuring that sites cannot silently query for digital credentials nor communicate with wallet providers without the user's active participation and confirmation of each action.
```

Comment:
```suggestion
          Require user interaction for requests , ensuring that sites cannot silently query for digital credentials nor communicate with wallet providers without the user's active participation and confirmation of each action.
```

Reply:
This should really be in the privacy considerations section, if not already there.